app/views/api/v2/smart_class_parameters/main.json.rabl

```diff
@@ -2,5 +2,5 @@ object @smart_class_parameter
 
 extends "api/v2/smart_class_parameters/base"
 
-attributes :description, :override, :parameter_type, :default_value, :use_puppet_default, :required, :validator_type, :validator_rule,
+attributes :description, :override, :parameter_type, :default_value, :hidden_value, :use_puppet_default, :required, :validator_type, :validator_rule,
```

Comment:
I think :hidden_value is definitely needed, when you consider a tool like Hammer is going to want to know and change a parameter's hidden_value attribute.

Whether the :default_value should be hidden or not in the API response, I'm unsure.  I think putting the onus on the API consumer to hide it is OK, since there might be cases (like in our UI) where having the non-hidden value available is still desired.